### PR TITLE
Fixes #198 dockerlinter checks files in COPY instruction

### LINKED ISSUE #198: [RFE] dockerfile linter should check files mentioned in COPY instructions
dockerfile linter (https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/dockerlinter.py) should check if files/directories mentioned in Dockerfile really exists in path.

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This Pull Request checks Dockerfile and files in COPY/ADD instructions

### CODE REVIEW

File: moduleframework/dockerlinter.py [hunk 1]
```diff
@@ -244,18 +245,40 @@ def check_clean_all_command(self):
...
         """
-        helpmd_present = False
+        files = []
         for instruction in [COPY, ADD]:
             try:
-                helpmd = [help for help in self.docker_dict[instruction] if "help.md" in help]
-                if helpmd:
-                    helpmd_present = True
+                # Get only source files, not the target
+                files.extend([x for x in self.docker_dict[instruction] if not x.startswith('/')])
```
Comment (TomasTomecek) [inline moduleframework/dockerlinter.py]: they can be relative, right? for ADD, those can also be http links

Comment (phracek) [reply moduleframework/dockerlinter.py]: I did not know it. Upss.

Comment (TomasTomecek) [reply moduleframework/dockerlinter.py]: https://docs.docker.com/engine/reference/builder/#add

File: moduleframework/dockerlinter.py [hunk 2]
```diff
@@ -244,18 +246,45 @@ def check_clean_all_command(self):
...
+        :return: True if all files/directories exist
+                 False otherwise
+        """
+        dir_name = os.getcwd()
+        files = self._get_copy_add_files(os.path.dirname(self.dockerfile))
+        f_exists = False
+        for f in files:
+            if f.startswith('http'):
+                f_exists = True
+            if os.path.exists(os.path.join(dir_name, f)):
```
Comment (lachmanfrantisek) [inline moduleframework/dockerlinter.py]: Is this necessary for the url? (when `f.startswith('http')`)

(`if` -> `elif`)

File: moduleframework/dockerlinter.py [hunk 3]
```diff
@@ -244,18 +246,48 @@ def check_clean_all_command(self):
...
+        :return: True if all files/directories exist
+                 False otherwise
+        """
+        dir_name = os.getcwd()
+        files = self._get_copy_add_files(os.path.dirname(self.dockerfile))
+        f_exists = False
+        for f in files:
+            if f.startswith('http'):
+                f_exists = True
+                continue
```
Comment (lachmanfrantisek) [inline moduleframework/dockerlinter.py]: OK, seems good.

**Pull request merged**